initrd-builder/initrd_builder.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 HyperHQ Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+script_name="${0##*/}"
+script_dir="$(dirname $(readlink -f $0))"
+
+if [ -n "$DEBUG" ] ; then
+	set -x
+fi
+
+SCRIPT_NAME="${0##*/}"
+INITRD_IMAGE="${INITRD_IMAGE:-kata-initrd.img}"
+
+die()
+{
+	local msg="$*"
+	echo "ERROR: ${msg}" >&2
+	exit 1
+}
+
+OK()
+{
+	local msg="$*"
+	echo "[OK] ${msg}" >&2
+}
+
+info()
+{
+	local msg="$*"
+	echo "INFO: ${msg}"
+}
+
+usage()
+{
+	error="${1:-0}"
+	cat <<EOT
+Usage: ${SCRIPT_NAME} [options] <rootfs-dir>
+	This script will create a Kata Containers initrd image file based on the
+	<rootfs-dir> directory.
+
+Options:
+	-h Show this help
+	-o path to generate image file ENV: INITRD_IMAGE
+
+Extra environment variables:
+	USE_DOCKER: If set will build image in a Docker Container (requries docker)
```

Comment:
Nit: "requires" (but I see that is also in the rootfs-builder and image-builder scripts too ;)

Reply:
Consider changing to:
USE_DOCKER: If set, the image builds in a Docker Container.  Setting this variable requires Docker.